### PR TITLE
Feature: Add backport and initializers

### PR DESCRIPTION
**Pull request by SatseAndrew**
This PR does two things:

- Adds backport for listRowSeparator. I tried some experiments to make this work on iOS 14. Let me know if you have a solution for this.
- Adds custom initializers for EdgeInsets.

### CODE REVIEW

**Review by kentrh — APPROVED** [review]
Looks good to me ^^

File: Sources/SATSCore/Extensions/SwiftUI/EdgeInsets.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    /// Convenience initializer to set bottom margin instead of specifying all of them
+    init(bottom: CGFloat) {
+        self.init(top: 0, leading: 0, bottom: bottom, trailing: 0)
+    }
+
+    /// Convenience initializer to set trailing margin instead of specifying all of them
+    init(trailing: CGFloat) {
+        self.init(top: 0, leading: 0, bottom: 0, trailing: trailing)
+    }
```
Comment (fespinoza) [inline Sources/SATSCore/Extensions/SwiftUI/EdgeInsets.swift:37]: are these really necessary?

- all
- vertical/horizontal
- top/horizontal/bottom

make sense to me on how we use the EdgeInsets.

but these singular ones are a bit weird to me... why not use `.padding(10, .top)` instead. I wouldn't use a `EdgeInset` in a single dimension case

**Review by fespinoza — APPROVED** [review]
It looks good!

just a minor question/nitpick

File: Sources/SATSCore/Extensions/SwiftUI/EdgeInsets.swift (hunk 1 above)
Comment (SatseAndrew) [reply Sources/SATSCore/Extensions/SwiftUI/EdgeInsets.swift:37]: That's true. I will remove the singular ones.

**Pull request merged**